docs/6.x/faq.md

```diff
@@ -156,3 +156,17 @@ the respective `tele build`, `gravity install` and `gravity upgrade`
 commands.
 
 See [Helm Integration](pack.md#helm-integration) for more details.
+
+## Changing Node Advertise Address
+
+Gravity provides a way to move a single-node cluster to a different node, for
+example to support a use-case of delivering a Gravity cluster as a part of the
+AMI/OVA/OVF appliance.
+
+See [Changing Node Advertise Address](/cluster/#changing-node-advertise-address) for more details.
+
+## Cluster Status
+
+Gravity provides the `gravity status` command to view [Cluster Status](/cluster/#cluster-status).
```

Comment:
These two urls you picked up from master should probably get the `.md` and stripping of the leading `/` treatment you do elsewhere.